debian-packaging.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
This script should probably go in either contrib/debian/ or scripts/

Reply:
Moved to scripts/ dir.